sign/impl.go

```diff
@@ -43,8 +44,9 @@ func (*defaultImpl) VerifyFileInternal(signer *Signer, path string) (*SignedObje
 	return signer.VerifyFile(path)
 }
 
-func (*defaultImpl) VerifyImageInternal(signer *Signer, reference string) (*SignedObject, error) {
-	return signer.VerifyImage(reference)
+func (*defaultImpl) VerifyImageInternal(ctx context.Context, images []string) (*SignedObject, error) {
+	v := verify.VerifyCommand{}
```

Comment:
To make the integration tests work, we'll need to pass the public key path here using `KeyRef`. Integration tests are not using keyless signing because that requires running in a trusted environment (e.g. Cloud Build).

Reply:
Makes sense @xmudrii . Will update